### PR TITLE
Add Contributing Section to Readme

### LINKED ISSUE #138: Contributing guide
@hgg had some issues setting up and contributing, and he volunteered to improve our docs and guides. I'm leaving it here just in case someone is thinking about doing it as well.

### PR DESCRIPTION
**Pull request by hgg**
Why:
* When doing my first contribution I found I did not have much
direction towards what is expected from contributors.

These changes address this issue by:
* Adding a contributing section to the readme
I considered other, more complex, alternatives but I believe for
now a simple section on the readme will suffice.

Issue #145 tackles another bump I found which I believe should be
solved not through documentation but through proper setup for
dev and test environments.

Closes #138

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,3 +115,4 @@ Still is released under the [ISC License](./LICENSE).
 [hex-badge]: https://img.shields.io/hexpm/v/still?style=flat-square
 [build-badge]: https://img.shields.io/github/workflow/status/still-ex/still/Elixir%20CI?style=flat-square
 [docs-badge]: https://img.shields.io/badge/-docs-informational?style=flat-square
+[issue-tracker]: https://github.com/still-ex/still/issues
```
Comment (frm) [inline README.md:118]: @gabrielpoca are we keeping Trello up to date? Does it make sense to list it instead?

Comment (gabrielpoca) [reply README.md:118]: @frm we are not, and to be honest I would rather keep everything in github. It's a bit disorganized, but I think it's working better than having another tool. What do you think?

**Comment by hgg** [comment]
I'll be adding a reference to imagemagick to this when I get the chance.

**Comment by gabrielpoca** [comment]
@hgg do you still want to merge it?

**Comment by hgg** [comment]
@gabrielpoca I do. I'll take care of it asap.

**Pull request merged**